flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/FlinkConfigOptions.java

```diff
@@ -40,4 +39,11 @@ private FlinkConfigOptions() {
           .intType()
           .defaultValue(100)
           .withDescription("Sets max infer parallelism for source operator.");
+
+  public static final ConfigOption<Boolean> TABLE_EXEC_ICEBERG_EXPOSE_SPLIT_LOCALITY_INFO =
+      ConfigOptions.key("table.exec.iceberg.expose-split-locality-info")
+          .booleanType()
+          .noDefaultValue()
```

Comment:
I think we should have a default value of `false` to be compatible with the existing behavior

Reply:
There was a [comment](https://github.com/apache/iceberg/pull/3817#discussion_r776882113) by @rdblue , about the default value.
> Is it possible to remove the default here? The purpose of this is to override the default (hdfs=on, everything else=off) and disable locality in splits, so I don't think that adding a default here makes sense.

in-add, If there is a `false` default value, may unable to determine whether `false` is set by user.

Reply:
oh. I got it now. We are retrieving the config as a boxed type `Boolean`. Yeah. In that case the default value is not needed.